src/modules/ResetPassword/ResetPassword.jsx

```diff
@@ -0,0 +1,117 @@
+                }
+            });
+    }
+
+    render() {
+        const { email, errorMsg, resetSuccess } = this.state;
+
+        return (
+            <div className="login-box">
+                <Card>
+                    <Card.Body className="login-card-body">
+                        {resetSuccess ? [
+                            <p key="help-text">
+                                Check your email for a link to reset your password.
+                            </p>,
+                            <a
+                                key="to-sign-in-link"
+                                className="btn btn-primary btn-block"
+                                href="/signin"
+                            >
+                                Return to sign in
+                            </a>,
+                        ] : [
+                            <p key="help-text" className="login-box-msg">
+                                You forgot your password?
+                                Here you can easily retrieve a new password.
+                            </p>,
+                            <form key="reset-password-form" onSubmit={this.onSubmit}>
+                                <Form.Group>
+                                    <InputGroup>
+                                        <Input
+                                            autoFocus
+                                            type="email"
+                                            name="email"
+                                            placeholder="Email"
+                                            onChange={this.onChange}
+                                            value={email}
+                                        />
+
+                                        <InputGroup.Append>
+                                            <InputGroup.Text>
+                                                <FontAwesomeIcon icon={faEnvelope} />
+                                            </InputGroup.Text>
+                                        </InputGroup.Append>
+                                        <div className="error invalid-feedback d-block">{errorMsg}</div>
+                                    </InputGroup>
+                                </Form.Group>
+
+                                <Button
+                                    variant="primary"
+                                    type="submit"
+                                    block
+                                >
+                                    Send password reset email
+                                </Button>
+                            </form>,
+                            <p key="sign-in-link" className="mb-0 mt-3">
+                                <a href="/signin">Sign in</a>
+                            </p>,
+                            <p key="sign-up-link" className="mb-0 mt-1">
+                                <a href="/signup">Register a new account</a>
+                            </p>,
```

Comment:
Getting to the page to get to the 'Sign up' page then doesn't make sense, so I suggest we will remove it as well.